Stream docs (#226)
svix/svix-docs: docs/stream/sinks/poller.mdx

---
title: Poller
---

# Polling Endpoints

Unlike other Sinks, Polling endpoints allow you to build clients to fetch events from the Stream directly, rather than having events pushed to you by Svix.

> Like all sinks, you can create the Poller the UI or [in the API](https://api.svix.com/docs#tag/Sink/operation/v1.stream.sink.create).

Reviewer: ```suggestion
Like all sinks, Polling Endpoints can be created in the Stream Portal or [in the API](https://api.svix.com/docs#tag/Sink/operation/v1.stream.sink.create).
```

Reviewer: same for the others
